### DISCUSSION TITLE
[LSP] Auto indexing for new files

### DISCUSSION THREAD

**Discussion by wojciech-kulik** [wojciech-kulik/xcodebuild.nvim#120] [Announcements]
@SolaWing is working on a feature that automatically indexes new files without rebuild. You can try it out now by using the version from the main branch and setting:

```lua
vim.env.XBS_FEAT_NEWFILE = 1
```

More details: https://github.com/SolaWing/xcode-build-server/issues/43#issuecomment-2027225353

> [!IMPORTANT]
> Make sure to rerun `xcode-build-server config` if you were using Homebrew version before.
> It will update the path to the binary in `buildServer.json`

**Comment by dudek-j** [comment]
Amazing, great job @SolaWing!

**Comment by wojciech-kulik** [comment]
@SolaWing, I think this flag is now enabled by default, am I correct? 
If so the step from this post is no longer needed.

**Reply by SolaWing** [reply]
yes，and you can disable it
